docs/source/data/data_catalog.md

```diff
@@ -36,20 +36,24 @@ shuttles:
 The dataset configuration in `catalog.yml` is defined as follows:
 1. The Top-level key is the dataset name used as a dataset identifier in the catalog - `shuttles`, `weather` in the example below.
 2. The next level includes multiple keys. The first one is the mandatory key, `type,` which defines the type of dataset to use.
-The rest of the keys are dataset properties and vary depending on the implementation.
-To get the extensive list of dataset properties, refer to {py:mod}`The kedro-datasets package documentation <kedro-datasets:kedro_datasets>` and navigate to the `__init__` method of the target dataset.
-3. Some dataset properties can be further configured depending on the libraries underlying the dataset implementation.
-In the example below, the configuration of the `load_args` property is defined by the `pandas` option for loading CSV files, while the configuration of the `save_args` property is defined by the `snowpark` `saveAsTable` method.
-To get the extensive list of dataset properties parameters, refer to {py:mod}`The kedro-datasets package documentation <kedro-datasets:kedro_datasets>` and navigate to the target property in the `__init__` definition for the dataset.
-For those properties we provide a reference to the underlying library configuration parameters. For example, under the `load_args` parameters section for [pandas.ExcelDataset](https://docs.kedro.org/projects/kedro-datasets/en/kedro-datasets-3.0.1/api/kedro_datasets.pandas.ExcelDataset.html) you may find a reference to the [pandas.read_excel](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.read_excel.html) method defining the full set of the parameters accepted.
+The rest of the keys are dataset parameters and vary depending on the implementation.
+To get the extensive list of dataset parameters, refer to {py:mod}`The kedro-datasets package documentation <kedro-datasets:kedro_datasets>` and navigate to the `__init__` method of the target dataset.
+3. Some dataset parameters can be further configured depending on the libraries underlying the dataset implementation.
+In the example below, the configuration of the `load_args` parameter is defined by the `pandas` option for loading CSV files, while the configuration of the `save_args` parameter is defined by the `snowpark` `saveAsTable` method.
```

Comment:
nit: I would also add the dataset names here, like `shuttles` when mentioning `load_args` and `weather` for `save_args` example and break this into two sentences as shorter sentences are easier to read.